src/client/scripts/esm/chess/logic/fivedimensional.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
Can you rename the file to include promote in its name since that's the only thing it does?